srcpkgs/libcec/template

```diff
@@ -39,6 +39,14 @@ cec-client_package() {
 		vmove usr/bin
 	}
 }
+python3-libcec_package() {
+	short_desc+=" - python bindings"
+	depends="python3 libcec>=${version}"
+
+	pkg_install() {
+		vmove usr/lib/python3.8
```

Comment:
There is futureproof `py3_lib` variable for that.

Reply:
true. I could use that too, but the value of `py3_lib` starts with a `/` which is not recommended for vmove. Afaics nothing in the system relies on the starting slash. remove it from the value too?

Reply:
fixed

Reply:
Right, values can drop slash then. #23073